docs/sdk_rest_api.md

```diff
@@ -62,4 +62,17 @@ Call when the GameServer session is over and it's time to shut down
 
 ```bash
 $ curl -d "{}" -H "Content-Type: application/json" -X POST http://localhost:59358/shutdown
-```
\ No newline at end of file
+```
+
+### GameServer
+
+⚠️⚠️⚠️ **/gameserver is currently a development feature and has not been released** ⚠️⚠️⚠️
+
+Call when you want to retrieve the backing `GameServer` configuration details
+
+- Path: `/gameserver`
+- Method: `GET`
+
+```bash
+$ curl -H "Content-Type: application/json" -X GET http://localhost:59358/gameserver
+```
```

Comment:
Should we show an expected response or link to the model?

Reply:
Good call. Added expected response.